elementary/monitor/api/tests/schema.py

```diff
@@ -50,6 +53,12 @@ class TestMetadataSchema(BaseModel):
     status: str
     test_created_at: Optional[str] = None
     days_diff: float
+    invocations_order: int
+    sample_data: Optional[Union[dict, List]] = None
+
+    # pydantic has a bug with Union fields. This is how to support it.
```

Comment:
This is not a bug, that's intended behavior.
I think we should remove this comment.